extensions/hp-extension/src/hp/hpHeart.ts

```diff
@@ -8,11 +8,26 @@ export default {
   editableBy: {},
   protocolMatchingRules: [
     {
-      id: 'LVFunction',
-      weight: 150,
-      attribute: 'StudyDescription',
+      id: 'MR',
+      weight: 50,
+      attribute: 'ModalitiesInStudy',
       constraint: {
-        contains: 'LV Function',
+        contains: 'MR',
+      },
+    },
+    {
+      id: 'SAX',
```

Comment:
Now have two study level constraints that say that to apply this hanging protocol, the study must have a series description with SAX and a series description with LAX.